deploy/README.md

```diff
@@ -93,6 +93,17 @@ the Ceph tree, such as
 
 But this is a matter of taste, and it really doesn't matter in the end.
 
+
+### Build requirements
+
+The following packages are required on openSUSE Tumbleweed for building the
+container:
+
+```
+    sudo zypper install git podman libvirt python38-devel python38-libvirt-python python38-pip buildah npm16
+    [sudo] pip install kcli
```

Comment:
this can very well be in its own venv (I do that, for instance), and I think that should be mentioned.